bundles/org.openhab.io.imperihome/README.md

```diff
@@ -496,6 +496,14 @@ Switch MyShutterStop "Stop shutter" ["iss:room:Test", "iss:type:DevSwitch"]  { c
 
 Implement a Rule listening for the ON command on the switch to handle the stop action yourself.
 
+## Tagging in OH3
+ To tag items in OH3 without using txt files, Tags need to be added like so:
```

Comment:
Binding documention should not go into steps on how to use openHAB, you should refer people to tag items and then if needed a link to the docs that cover that section. 
This is to stop documentation getting outdated when the UI changes.

https://www.openhab.org/docs/configuration/items.html#tags

It appears that the link I gave above does not cover how to do in the main UI, so what you have done would be a great PR to add your information to that part of the docs and then you and others can refer to the section.

Does this make sense? It is so we only need to change 1 area in the docs if the UI changes instead of having to review every single binding readme.

Reply:
> Does this make sense?

Yes, this makes a lot of sense! Information that is not specific to a binding should be added to the general documentation only. I'd thus suggest to close this PR and would be happy to see a PR against openhab-docs!